llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -596,13 +598,15 @@ class VPRecipeWithIRFlags : public VPSingleDefRecipe {
     DisjointFlagsTy(bool IsDisjoint) : IsDisjoint(IsDisjoint) {}
   };
 
+  struct NonNegFlagsTy {
+    char NonNeg : 1;
```

Comment:
nit: (to keep with convention in this class).
```suggestion
    char IsNonNeg : 1;
```

Also, can this just be a `bool` rather than a bitfield struct? (I don't expect any other NonNegFlags to be added)